### ISSUE TITLE
PLF12 problem

### ISSUE THREAD

**Issue by Moosbrugger** [KaufHA/common#16]
I had the plug working at one point but then I changed something.  Now it doesn't seem to want to do anything -- I can't reach it from any url (the one assigned to it by my network, which it _is_ logged into or, 192.168.1.4) .  Holding the button for 30+ seconds doesn't seem to reset it.  (I also tried using the /reset command from the earlier plug.)  And I see no LED response.  If I then unplug it & plug it back in I get 3-4 very slow red blinks & then they stop. But nothing's changed.

**Comment by bkaufx** [comment]
Can you see it connected to your wifi network?

**Comment by Moosbrugger** [comment]
Yes: it's connected to my LAN (as said in my original post -- I can't reach it from any url (the one assigned to it by my network, which it is logged into or, 192.168.1.4). Sorry if I wasn't clear.

**Comment by bkaufx** [comment]
Can you flash it via OTA using ESPHome dashboard?

**Comment by Moosbrugger** [comment]
My ESP dashboard can't see it.  
0) No: it doesn't show up on the ESPHome dashboard; even after restarting ESP; even after rebooting Home Assistant
1) it has a LAN address
2) can't connect to to it via the ip address; /rest & /clear don't do anything
3) holding the button down for 30 seconds does nothing
4) curl commands can't connect
